core/src/main/scala/ai/h2o/sparkling/backend/converters/DataTypeConverter.scala

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Just thinking whether we should merge https://github.com/h2oai/sparkling-water/blob/master/core/src/main/scala/ai/h2o/sparkling/backend/utils/ConversionUtils.scala with this class?

Reply:
Ok, will move the logic to `ConversionUtils`

Reply:
I meant merge ConverstionUtils with DataTypeConverter, that might make more sense, WDYT?

Reply:
> Ok, will move the logic to `ConversionUtils`
I meant to move all methods from `DataTypeConverter` to `ConverstionUtils` and remove `DataTypeConverter`.

Reply:
:D I thought remove `ConverstionUtils` and move to `DataTypeConverter` to other conversion methods. But might be just personal taste

Reply:
ok, np. Will move it to `DataTypeConverter`